Assets/CodeBlockFamilyPositions.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CodeBlockFamilyPositions : MonoBehaviour
+{
+    // Start is called before the first frame update
```

Comment:
Is this file needed? Delete if not or remove all uneccesary stuff in file (also needs namespace)